pkg/qdr/qdr.go

```diff
@@ -836,9 +838,68 @@ func (a *BridgeConfig) Difference(b *BridgeConfig) *BridgeConfigDifference {
 		HttpConnectors: a.HttpConnectors.Difference(b.HttpConnectors),
 		HttpListeners:  a.HttpListeners.Difference(b.HttpListeners),
 	}
+
+	result.AddedSslProfiles = checkAddedSslProfiles(result)
```

Comment:
I wonder if it would result in simpler, more intuitive code, if we just go through all endpoints for original config and create a map of the ssl profile names, then repeat that for the new config, creating a separate map, then compare the two maps to get the added and deleted ssl profile names?

Reply:
Thanks for this suggestion, I think that it is more clear now; let me know if the solution works for you.